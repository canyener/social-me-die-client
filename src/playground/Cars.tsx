import React from 'react'

import { cars } from '././cars-data'
import CarItem from './CarItem'

const Cars = () => (
  <div>
    <ul>
      {cars.map((car) => <CarItem key={car.model} car={car} /> )}
    </ul>
  </div>
)

export default Cars