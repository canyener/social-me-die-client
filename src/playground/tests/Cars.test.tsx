import React from 'react'
import { shallow } from 'enzyme' 

import Cars from '../Cars'

test('Should render Cars correctly', () => {
  const wrapper = shallow(<Cars />)

  expect(wrapper).toMatchSnapshot()
})