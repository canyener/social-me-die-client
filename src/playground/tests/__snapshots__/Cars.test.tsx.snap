// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Cars correctly 1`] = `
<div>
  <ul>
    <CarItem
      car={
        Object {
          "color": "blue",
          "model": "BMW",
        }
      }
      key="BMW"
    />
    <CarItem
      car={
        Object {
          "color": "red",
          "model": "Mercedes",
          "topSpeed": 100,
        }
      }
      key="Mercedes"
    />
  </ul>
</div>
`;
